api.swagger.yaml

```diff
@@ -6497,12 +6594,14 @@ components:
             - Note
             - Done
             - News / Event
-            - Suggestion
+            - Idea
```

Comment:
These would align to Akagera data, but not what is setup on Live. So we shouldn't make this change.

Reply:
I put it back